notifier/chatters.rst

```diff
@@ -218,6 +218,92 @@ to add `message options`_::
 
     $chatter->send($chatMessage);
 
+Adding text to a Microsoft Teams Message
+-----------------------------------------
+
+With a Microsoft Teams, you can use the simple ChatMessage::
+
+    use Symfony\Component\Notifier\Bridge\MicrosoftTeams\MicrosoftTeamsTransport;
+    use Symfony\Component\Notifier\Message\ChatMessage;
+
+    $chatMessage = (new ChatMessage('Contribute To Symfony'))->transport('microsoftteams');
+    $chatter->send($chatMessage);
+
+The result will be something like:
+
+.. image:: /_images/notifier/microsoft_teams/message.png
+   :align: center
+
+Adding Interactions to a Microsoft Teams Message
+-----------------------------------------
```

Comment:
There should be the same length. 

@OskarStark, shouldn't DOCtor find this?

Reply:
Fixed

Reply:
> @OskarStark, shouldn't DOCtor find this?

AFAIK there is no such rule, because Sphinx allows this 🤔